packages/request-client.js/src/api/request-network.ts

```diff
@@ -163,7 +169,14 @@ export default class RequestNetwork {
     );
 
     // create the request object
-    const request = new Request(this.requestLogic, requestId, paymentNetwork, this.contentData);
+    const request = new Request(
+      this.requestLogic,
+      requestId,
+      paymentNetwork,
+      this.contentData,
+      undefined,
```

Comment:
This undefined here looks weird :/
I'm ok with leaving it for now, but we have to change it in the future (maybe add it to the options?)

Reply:
I put it this way because I was thinking that it was a breaking change. but the request constructor is not meaned to be public. so I can arrange this !